Packs/CTIX/Integrations/CTIXv3/CTIXv3.py

```diff
@@ -1924,6 +1968,102 @@ def make_request(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
             ]
 
 
+def cve_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    page = 1
+    page_size = 15
+    params = {"page": page, "page_size": page_size}
+    cve = args["cve"]
+    if isinstance(cve, str):
```

Comment:
Please use argToList on cve instead of split

Reply:
Thanks! I made the change as you suggested.